pkg/accounts/adaptor/repository/prisma.ts

```diff
@@ -288,6 +288,7 @@ export class PrismaAccountFollowRepository implements AccountFollowRepository {
     targetID: AccountID,
   ): Promise<Result.Result<Error, void>> {
     try {
+      // ToDo: use hard delete (Because it is a composite primary key, it will never be possible to follow it back again.)
```

Comment:
```suggestion
      // ToDo: Should replace with a hard delete. It can't follow it back again due to a composite primary key.
```
ちょっと文意が読み取りにくいので, ハード削除にしたほうが良いという書き方にしてはどうでしょうか